arbnode/maintenance.go

```diff
@@ -193,9 +189,19 @@ func (mr *MaintenanceRunner) maybeRunScheduledMaintenance(ctx context.Context) t
 		return time.Minute
 	}
 
-	err := mr.attemptMaintenance(ctx)
+	shouldTriggerMaintenance, err := mr.exec.ShouldTriggerMaintenance().Await(mr.GetContext())
```

Comment:
The existing maintenance runner had a time of day for config and tried to execute maintenance once a day at that time.
So current code will ask "shouldtriggermaintenance" once a day, which is not what we want.
If we just want to poll cexecution node, it should be done with some interval (default around 1 minute) with. a "normal" time-interval config.